sources/safe/arbitrum/safe_arbitrum_sources.yml

```diff
@@ -29,4 +29,32 @@ sources:
             description: "Address of created Safe proxy"
           - &singleton
             name: singleton
-            description: "Singleton"
\ No newline at end of file
+            description: "Singleton"
+
+  - name: gnosis_safe_arbitrum
+    freshness:
+      warn_after: { count: 12, period: hour }
+    tables:
+      - name: ProxyFactoryv_1_4_1_evt_ProxyCreation
+        loaded_at_field: evt_block_time
+        description: "Safe v1.4.1 factory ProxyCreation events"
+        columns:
+          - *contract_address
+          - &evt_block_number
+            name: evt_block_number
+            description: "Event block number"
+          - &evt_block_time
+            name: evt_block_time
+            description: "Event block time"
+          - &evt_index
+            name: evt_index
```

Comment:
Actually for all columns already defined in the same `.yml` file, we can use `*` syntax.
Otherwise it may cause compilation error.